css/general.css

```diff
@@ -1,8 +1,55 @@
+
 body {
+  max-width: 1200px;
+  margin: 0 auto;
   font-size: var(--text-size-default);
   font-family: var(--text-family);
-  color: var(--main-text-color);
   background-color: var(--main-bg-color);
+  background-image: url(../img/wall.jpg);
+
+  min-height: 100vh;
+}
+.main-navigation {
+  background-color: blue;
+  position: relative;
+  text-align: right;
+
+
 }
+.main-navigation, a {
```

Comment:
This is wrong. You are setting all `<a>` tags on the page to have the following css properties and values, which should not be the case and might created overflowing elements. Keep the styles only for **.main-navigation a**.